### ISSUE TITLE
msconvert  filter "demultiplex optimization=overlap_only"

### ISSUE THREAD

**Issue by jj-umn** [galaxyproteomics/tools-galaxyp#464]
@chambm  DIA analysis for Skyline and EncyclopeDIA suggests converting Thermo raw files to mzML with --filter "demultiplex optimization=overlap_only".  
Is that something that is or can be available in the Galaxy tool.
Thanks, 
JJ

**Comment by chambm** [comment]
It's not currently available but it should be easy to add. I think this is only recommended for overlapping DIA windows though. Adjacent windows don't need this.

**Comment by jj-umn** [comment]
@chambm  Yes.   This is the advice from Brian Searle for using EncyclopeDIA for DIA:

You should use MSConvert to deconvolute all DIA raw files. You need to use these options:
msconvert.exe --zlib --64 --mzML --simAsSpectra --filter "peakPicking true 1-" --filter "demultiplex optimization=overlap_only" *.raw 
Best,
Brian
